.travis.yml

```diff
@@ -98,16 +98,17 @@ install:
 - pushd $TRAVIS_BUILD_DIR
 - source scripts/setup.sh || exit 1
 - source scripts/setup_gae.sh || exit 1
+- bash scripts/install_third_party.sh
```

Comment:
Why did we need to add this?

Reply:
Earlier this line ` if [ "$RUN_LINT" == 'true' ]; then bash scripts/install_third_party.sh; python scripts/third_party_size_check.py; python scripts/pre_commit_linter.py --path=.; fi` was taking care of installing the third party libraries. Since we have now commented this part out, we need to install the third party lib, nevertheless. Therefore, I have placed this here -- independent of all checks, running or not.

Reply:
Oh I see, thanks! Please could you remove this line from other places then, even if commented out -- e.g. remove it from line 108? That way the code there is clean if we ever need to switch back.

Reply:
Done!